dhall/src/Dhall/Syntax.hs

```diff
@@ -196,33 +217,34 @@ instance Pretty Var where
 --
 -- … will be instantiated as follows:
 --
--- * @bindingSrc0@ corresponds to the @A@ comment.
+-- * @bindingComment0@ corresponds to the @A@ comment.
 -- * @variable@ is @"x"@
--- * @bindingSrc1@ corresponds to the @B@ comment.
+-- * @bindingComment1@ corresponds to the @B@ comment.
```

Comment:
Don't forget to also document what `variableSrc` would correspond to in this example